src/datapump/jobs/geotrellis.py

```diff
@@ -734,6 +734,9 @@ def _run_job_flow(self, name, instances, steps, applications, configurations):
 
     @staticmethod
     def _instances(worker_instance_count: int) -> Dict[str, Any]:
+        core_count = round(worker_instance_count / 8)
```

Comment:
Perhaps `math.ceil` would be better here?   That way there will always be at least one core instance, even if `worker_instance_count  < 4`, and  10 would have  2, etc.

Reply:
Also, very minor but elsewhere this variable is called `worker_count`, while it's called `worker_instance_count` here.  Might be good to be consistent just to indicate that we're talking about the same value typically.